### ISSUE TITLE
Inform that HME has been used for that website already

### ISSUE THREAD

**Issue by smoochy** [dedoussis/icloud-hide-my-email-browser-extension#29]
Hi again :)

I wanted to suggest a feature for the forgetful users of us.

When I am opening the extension, it is offering to create an HME for that website. But it is not recognizing, that I already did that in the past. I would need to manually switch to the "Manage Emails" tab, search for that website if it exists or not.

It would be helpful, if there is a little banner telling me, that I already created an HME for that specific domain. It should still offer me to create a new one though.
The banner should also recognize, when I created an HME for _https://domain. com__, but now accessing _https://sub.domain.com_

Kind Regards

**Comment by dedoussis** [comment]
Hi @smoochy! I've thought about this feature (which Safari also provides) but I've omitted it in purpose because I didn't want the scope of the extension to overlap with the scope of a password manager. Password managers remind users of their existing emails, upon focusing on an email input element. That's one of the core features of every password manager. On the other hand, the primary concern of the HME extension is the generation of new addresses. I think it's better to keep these two concerns separate, and advise users to rely on password managers for the purposes of remembering existing addresses, but I'm happy to be challenged on that opinion.

**Comment by dedoussis** [comment]
That said, I think it would be useful to add a small hint in the extension icon, whenever the user visits a website for which an HME already exists. Something like a blue dot next to the apple.

**Comment by smoochy** [comment]
Well, I think it is important to inform the forgetful user, that an alias had been created already for that domain.
If you think, a little banner on the "Create an address for" page would be too intrusive, an icon, as proposed by you, would be a good and decent trade-off.

**Comment by felix-exon** [comment]
I agree with @smoochy. I think for most user's the use-case is ONE email per service.
I would also love an option to omit the html injection for all sites i have already used hme for.
For me it happens alot, that I accidentally click on it and create another alias because sometimes hme is faster than 1password.

**Comment by dedoussis** [comment]
> I would also love an option to omit the html injection for all sites i have already used hme for

@felix-exon this is great feedback, thank you! I'll look into adding this whenever I get the time to work on this issue.

If you find this to be too annoying you can navigate to the [options](https://github.com/dedoussis/icloud-hide-my-email-browser-extension?tab=readme-ov-file#options) of the extension to disable the autofill button and rely solely on the context menu.
